### DISCUSSION TITLE
[Keyball61]トラックボール側（右で使用）ジャンパ部分のパッド剥離

### DISCUSSION THREAD

**Discussion by air-cocoa** [Yowkees/keyball#208] [Q&A]
ジャンパ部分のブリッジに失敗してパッドが剥離してしまいました。
回路図の見方がわからず、直接銅線等でつなげて動作するようにするにはどのようにすればいいか
お手数をお掛けしますが、ご教授よろしくお願いいたします。

過去のDiscussionで同様の質問をしている方の回答内容と同じことを行えば解決できるのでしょうか？
こちらの方はKeyball39での問題でしたので確認させてください。
https://github.com/Yowkees/keyball/discussions/187#discussioncomment-4753833

![SC](https://user-images.githubusercontent.com/129810557/229683457-bf82abe1-baac-4d73-bad6-1c022a2f0b64.png)

**Comment by Yowkees** [comment]
![61jamper_fix](https://user-images.githubusercontent.com/84209772/229946236-61a77e45-9fa3-4f9e-bdea-66346968fb06.jpg)
ご連絡ありがとうございます。状況、了解しました。
赤丸をつけた２点を導線で接続してください。
なお、この作業は４Ｐピンソケット、タクトスイッチを半田付けした後に行ってください。

また、赤丸をつけたスルーホールに半田が付着しているようです。ここに半田ごてで熱を加えて溶かしながら４Ｐピンソケットを差し込めば挿入できますので、作業お願いいたします。

**Comment by Yowkees** [comment]
回路図よりも下記にて公開している基板図をみていただければ、どことどこが繋がっているかだけ見ればよいので、理解しやすいと思います。
https://github.com/Yowkees/keyball/tree/main/keyball61/design_data

**Comment by air-cocoa** [comment]
ご回答ありがとうございます。
上記対応で作成進めて行きます。
基盤図の見方調べてみます。
